EIPS/eip-681.md

```diff
@@ -0,0 +1,81 @@
+## Preamble
+
+    EIP: 681
+    Title: URL Format for Transaction Requests
+    Author: Daniel A. Nagy <daniel@ethereum.org>
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Created: 2017-08-01
+    Requires: 20, 137
+
+## Simple Summary
+A standard way of representing various transactions, especially payment requests in Ethers and ERC #20 tokens as URLs.
+
+## Abstract
+URLs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very loosely coupled applications. A standardized URL format for payment requests allows for instant invocation of the user's preferred wallet application (even if it is a webapp or a swarm đapp), with the correct parameterization of the payment transaction only to be confirmed by the (authenticated) user.
+
+## Motivation
+The convenience of representing payment requests by standard URLs has been a major factor in the wide adoption of Bitcoin. Bringing a similarly convenient mechanism to Ethereum would speed up its acceptance as a payment platform among end-users. In particular, URLs embedded in broadcast Intents are the preferred way of launching applications on the Android operating system and work across practically all applications. Desktop web browsers have a standardized way of defining protocol handlers for URLs with specific protocol specifications. Other desktop applications typically launch the web browser upon encountering a URL. Thus, payment request URLs could be delivered through a very broad, ever growing selection of channels.
+
+This specification supersedes ERC #67, which is a URL format for representing arbitrary transactions in a low-level fashion. This ERC focuses specifically on the important special case of payment requests, while allowing for other, ABI-specified transactions.
+
+## Specification
+
+### Syntax
+Payment request URLs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+
+    request                 = "ethereum" ":" [ "pay-" ]target_address [ "@" chain_id ] [ "/" function_name ] [ "?" parameters ]
+    target_address          = ethereum_address
+    chain_id                = 1*DIGIT
```

Comment:
Why is chain_id one digit?  That means there can only be 10 ethereum based chains ever, and this can't be used for test networks (which have chain IDs that are 4+ digits).  This really feels like it should be `DIGIT *[DIGIT]` (or whatever the notation is for unbounded).

Reply:
@MicahZoltu If I take a look at the definition of `number` I would guess this means `at least one digit`